### PR TITLE
SDK refacto + First coston integration

### PR DESCRIPTION
**Pull request by antiyro**
Here is the clean PR that goes with the PR of the interface and the one of sdk (2.0.1). This one contains :

- The refacto of the sdk + link of the constants of the sdk
- The first integration of COSTON

this PR depends on https://github.com/pangolindex/sdk/pull/27

### CODE REVIEW

File: src/components/CurrencyLogo/index.tsx [hunk 1]
```diff
@@ -32,6 +32,8 @@ export default function CurrencyLogo({
     return <AvaxLogo size={`${size}px`} />;
   } else if (deepEqual(currency, CAVAX[ChainId.WAGMI])) {
     return <WgmLogo size={`${size}px`} />;
+  } else if (deepEqual(currency, CAVAX[ChainId.COSTON])) {
+    return <WgmLogo size={`${size}px`} />;
```
Comment (SarjuHansaliya) [inline src/components/CurrencyLogo/index.tsx:36]: I think we need to add coston/flare logo here

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pangolindex/components",
-  "version": "2.0.3",
+  "version": "2.0.4",
```
Comment (SarjuHansaliya) [inline package.json:3]: we dont need to update version manually, it will be automatically updated via github actions

File: src/components/SwapWidget/TradeOption/index.tsx [hunk 3]
```diff
@@ -18,7 +22,7 @@ const TradeOption: React.FC<Props> = ({ swapType, setSwapType, isLimitOrderVisib
           <Text color="text1" fontSize={24} fontWeight={500}>
             Trade
           </Text>
-          {isLimitOrderVisible && (
+          {isLimitOrderVisible && CHAINS[chainId].mainnet && (
```
Comment (SarjuHansaliya) [inline src/components/SwapWidget/TradeOption/index.tsx:25]: we dont need `CHAINS[chainId].mainnet` here because `isLimitOrderVisible` already being handled on interface side.

File: yarn.lock [hunk 4]
```diff
@@ -2,6 +2,19 @@
 # yarn lockfile v1
 
 
+"@pangolindex/sdk@10.0.12":
+  version "10.0.12"
```
Comment (SarjuHansaliya) [inline yarn.lock:6]: there is some issue here, version is not proper i think

**Review by SarjuHansaliya — CHANGES_REQUESTED** [review]

File: src/components/SwapWidget/TradeOption/index.tsx (hunk 3 above)
Comment (antiyro) [reply src/components/SwapWidget/TradeOption/index.tsx:25]: The button still appears on testnets on the interface

Comment (SarjuHansaliya) [reply src/components/SwapWidget/TradeOption/index.tsx:25]: you need to pass `CHAINS[chainId].mainnet` from interface because swap component is global component and can be used by any project so we dont want to include condition that affects other project

Comment (antiyro) [reply src/components/SwapWidget/TradeOption/index.tsx:25]: Okk we'll do it from interface

File: yarn.lock (hunk 4 above)
Comment (antiyro) [reply yarn.lock:6]: done ✅

File: package.json (hunk 2 above)
Comment (antiyro) [reply package.json:3]: ok ✅

File: src/components/SwapWidget/TradeOption/index.tsx [hunk 5]
```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { Box, Text, ToggleButtons } from '../../';
 import { SwapWrapper } from './styled';
+import { CHAINS } from '@pangolindex/sdk'
```
Comment (SarjuHansaliya) [inline src/components/SwapWidget/TradeOption/index.tsx:4]: unused variable

File: src/constants/lists.ts [hunk 6]
```diff
@@ -1,9 +1,11 @@
 // the Pangolin Default token list lives here
-export const PANGOLIN_LIST = 'https://raw.githubusercontent.com/pangolindex/tokenlists/main/pangolin.tokenlist.json';
+export const AVALANCHE_LIST = 'https://raw.githubusercontent.com/pangolindex/tokenlists/main/pangolin.tokenlist.json';
+export const WAGMI_LIST = 'https://raw.githubusercontent.com/pangolindex/tokenlists/main/wgm.tokenlist.json'
+export const COSTON_LIST = 'https://raw.githubusercontent.com/pangolindex/tokenlists/main/cflr.tokenlist.json'
```
Comment (SarjuHansaliya) [inline src/constants/lists.ts:4]: this tokenlist doesn't exist

File: yarn.lock [hunk 7]
```diff
@@ -1,12604 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (SarjuHansaliya) [inline yarn.lock:1]: why is this file removed ? please revert back

File: package.json [hunk 8]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pangolindex/components",
-  "version": "2.0.8",
+  "version": "2.0.3",
```
Comment (SarjuHansaliya) [inline package.json:3]: please don't update this version, this is automatic updation

**Review by SarjuHansaliya — CHANGES_REQUESTED** [review]

File: src/components/SwapWidget/TradeOption/index.tsx [hunk 9]
```diff
@@ -9,6 +11,8 @@ interface Props {
 }
 
 const TradeOption: React.FC<Props> = ({ swapType, setSwapType, isLimitOrderVisible }) => {
+  const chainId = useChainId()
```
Comment (SarjuHansaliya) [inline src/components/SwapWidget/TradeOption/index.tsx:14]: this is unused variable

**Review by SarjuHansaliya — CHANGES_REQUESTED** [review]

File: package.json (hunk 8 above)
Comment (antiyro) [reply package.json:3]: Yeah I am sorry, I tought that I should keep the version of the PR, so you want me to put 2.0.8? Or should I leave it as .3 and it will update?

Comment (SarjuHansaliya) [reply package.json]: yes 2.0.8 is fine, thanks a lot
